pkg/cri/sbserver/container_create_linux_test.go

```diff
@@ -1733,28 +1733,50 @@ func TestCDIInjections(t *testing.T) {
 	for _, test := range []struct {
 		description   string
 		cdiSpecFiles  []string
+		cdiDevices    []*runtime.CDIDevice
 		annotations   map[string]string
 		expectError   bool
 		expectDevices []runtimespec.LinuxDevice
 		expectEnv     []string
 	}{
-		{description: "expect no CDI error for nil annotations"},
+		{description: "expect no CDI error for nil annotations",
+			cdiDevices: []*runtime.CDIDevice{},
+		},
+		{description: "expect no CDI error for nil CDIDevices",
```

Comment:
is this duplicate of the test case on line 1748?

Reply:
They're almost the same, but slightly different. This one tests nil annotations and the one on line 1748 tests empty annotations.

Reply:
The test case _content_ appears to be identical to me, can they be unified?

Reply:
@TBBle Thank you for the review!
Done.